.github/workflows/docker-images-nightly.yml

```
@@ -53,6 +53,13 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3.0.0
 
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3.0.0
+        with:
```

```suggestion
      - name: Login to GitHub Container Registry
        if: github.event.inputs.tag == 'nightly'
        uses: docker/login-action@v3.0.0
        with:
```

and maybe we can move this step just above the `Push image to GHCR` step.